docs/import/manage_layers.js

```diff
@@ -402,3 +404,17 @@ function createOptionFrom(innerTextAndValue) {
       .val(innerTextAndValue)
       .prop('id', innerTextAndValue);
 }
+
+/**
+ * Adds a non-ee layer to the map. 
+ */
+function addNonEELayer() {
+  const type = parseInt($('#non-eelayer-type').val());
+  const urls = $('#non-eelayer-urls').val().split('\n');
+
+  processNonEeLayer($('#non-eelayer-name').val(), type, urls);
```

Comment:
Might be nice to return this promise so we know when it's added (and for easier testing)